addon/components/ember-node/embedded-editor.ts

```diff
@@ -97,31 +93,45 @@ export default class EmbeddedEditor extends Component<Args> {
     return this.args.controller;
   }
 
-  get schema() {
-    return new Schema({
-      nodes: {
-        doc: {
-          content: 'block+',
-        },
-        paragraph,
-        repaired_block,
-        placeholder,
-
-        text,
+  get plugins() {
+    return this.args.plugins || [];
+  }
 
-        hard_break,
-        block_rdfa,
-        invisible_rdfa,
-      },
-      marks: {
-        inline_rdfa,
-        link,
-        em,
-        strong,
-        underline,
-        strikethrough,
-      },
-    });
+  get keymap() {
```

Comment:
Maybe we should let `this.args.keymap` override the whole keymap so it is fully customizable?

Reply:
it is done this way because undo/redo should work on the "outer state" and really the embedded-editor should not care about what undo/redo does. 

You could remove `...basemap`, so there only the undo/redo overrides are done. That might make more sense (and baseMap should really be inside keymap.ts, so a node using embedded-editor can choose to use baseMap and change whatever it wants from it).